### PR TITLE
Fix the incorrect type of the min, max and value attributes passed to the Input

### PR DESCRIPTION
**Pull request by vitoriaHeliane**
#### What is the purpose of this pull request?
Fix the incorrect type of the min, max and value attribute passed to the Input, because it was generating some errors in the console. 
captura de tela de 2018-07-17 11-09-54

#### How should this be manually tested?
https://waza--storecomponents.myvtex.com/

#### Types of changes
- [X] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/form/BaseInput.js [hunk 1]
```diff
@@ -39,17 +39,17 @@ const BaseInput = props => {
       error={!!currentError}
       errorMessage={currentError}
       helpText={schema.description}
-      label={<FormattedMessage id={label}/>}
-      max={max}
-      min={min}
+      label={label}
+      max={max ? `${max}` : max}
```
Comment (alcar) [inline react/components/form/BaseInput.js]: I think it'd be better to just use `` max && `${max}` ``, as the fallback value is already `undefined` (that also applies to the other changes)

Comment (vitoriaHeliane) [reply react/components/form/BaseInput.js]: Done!

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,12 +9,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+### Fixed
+- Incorrect type of the max, min and value attibutes passed to the `Input`.
+
 ## [1.12.5] - 2018-07-19
 ### Fixed
 - Remove `template` and `site` scope conditions.
 
 ## [1.12.4] - 2018-07-18
 ### Fixed
+- Incorrect type of the max, min and value attibutes passed to the `Input`.
```
Comment (andreldsa) [inline CHANGELOG.md]: Duplicated

**Comment by lucasecdb** [comment]
Closing in favor of #66

**Pull request closed without merging**